package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "raygun",
-  "description": "Raygun.io package for Node, written in TypeScript",
-  "version": "0.13.2",
+  "description": "Raygun package for Node.js, written in TypeScript",
+  "version": "0.14.0",
   "homepage": "https://github.com/MindscapeHQ/raygun4node",
   "author": {
     "name": "MindscapeHQ",
```

Comment:
Please update all "MindscapeHQ" references to "Raygun" :-)

Reply:
It might be worthwhile checking if there's a connection between `author.name` in package.json and the account that RG is using to publish to npm. I'm not sure if there's a requirement for them to match maybe?

Reply:
Author name doesn't appear on npm even https://www.npmjs.com/package/raygun

The documentation doesn't mention any requirement for them to match https://docs.npmjs.com/cli/v6/configuring-npm/package-json#people-fields-author-contributors

It's a quick change nevertheless so I can do that.

Reply:
PR: https://github.com/MindscapeHQ/raygun4node/pull/185